Reference.md

```diff
@@ -329,6 +342,8 @@ buffer are Emacs buttons -- which you may navigate among using
 TAB, and activate using RET -- for [`racket-visit-definition`](#racket-visit-definition) and
 [`racket-doc`](#racket-doc).
 
+(fn &optional PREFIX)
```

Comment:
I ran `make doc` and this appeared, which I'm not sure is intended. But I'm leaving it as this file is auto-generated.

Reply:
The new command I added is there. I’m just confused that this appeared as well.

Reply:
Ok I've attempted to debug this and I'm not clever enough. I'll let you re-run `make doc` and see if that removes the line.

Reply:
> The new command I added is there.
Ah! I misunderstood the diff snippet. (I've forgotten to add a command to that list. So I just assumed, without looking more carefully.)